target/linux/ath79/generic/base-files/etc/board.d/03_gpio_switches

```diff
@@ -82,7 +82,8 @@ ubnt,nanostation-m)
 ubnt,nanostation-m-xw)
 	ucidef_add_gpio_switch "poe_passthrough" "PoE Passthrough" "2"
 	;;
-ubnt,acb-isp)
+ubnt,acb-isp|\
+ubnt,acb-ac)
```

Comment:
This still needs to be updated.

Reply:
thank you, sorted and used full names here.